test/e2e/specs/beforesuite.go

```diff
@@ -18,10 +18,7 @@ import (
 // The client will be configured to work either against the real, fake or admin apis
 type rpFocus string
 
-var (
-	fakeRpFocus = rpFocus(regexp.QuoteMeta("[Fake]"))
-	realRpFocus = rpFocus(regexp.QuoteMeta("[Real]"))
-)
+var fakeRpFocus = rpFocus(regexp.QuoteMeta("[Fake]"))
```

Comment:
I think because we have only fake tests now we can remove `[Fake]` at all. Less tags - easier to read and manage code

Reply:
requirements.md in `docs/e2e/requirements.md` need to be updated too